python/py-spdx-lookup/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-spdx-lookup
+version             0.3.2
+revision            0
+
+maintainers         nomaintainer
+description         SPDX license list query tool
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/bbqsrc/spdx-lookup-python
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             2-clause BSD License
```

Comment:
This is not a string that MacPorts would recognise and associate with the licence you had in mind.

See https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

Now, ironically, I would say that we would need to improve support for precisely SPDX in MacPorts.
For the start we would probably need a mapping between whatever `upt` says a licence name is, and the keyword that's known to MacPorts.

Reply:
For now I am editing it manually.

Finally will add a conversion script to Macports backend.